docs/getting-started-guides/coreos/bare_metal_calico.md

```diff
@@ -44,81 +39,186 @@ Specifically, this guide will have you do the following:
 
 ## Prerequisites
 
-1. At least three bare-metal machines (or VMs) to work with. This guide will configure them as follows:
-  - 1 Kubernetes Master
-  - 2 Kubernetes Nodes
-2. Your nodes should have IP connectivity.
+-   At least three bare-metal machines (or VMs) to work with. This guide will configure them as follows:
+    - 1 Kubernetes Master
+    - 2 Kubernetes Nodes
+-   Your nodes should have IP connectivity to each other and the internet.
 
 ## Cloud-config
 
 This guide will use [cloud-config](https://coreos.com/docs/cluster-management/setup/cloudinit-cloud-config/) to configure each of the nodes in our Kubernetes cluster.
 
 We'll use two cloud-config files:
-- `master-config.yaml`: Cloud-config for the Kubernetes master
-- `node-config.yaml`: Cloud-config for each Kubernetes node
+- `master-config.yaml`: cloud-config for the Kubernetes master
+- `node-config.yaml`: cloud-config for each Kubernetes node
 
 ## Download CoreOS
 
-Let's download the CoreOS bootable ISO.  We'll use this image to boot and install CoreOS on each server.
+Download the CoreOS bootable ISO from the [CoreOS website](https://coreos.com/docs/running-coreos/platforms/iso/).
 
-```
-wget http://stable.release.core-os.net/amd64-usr/current/coreos_production_iso_image.iso
-```
+## Configure the Kubernetes Master
 
-> You can also download the ISO from the [CoreOS website](https://coreos.com/docs/running-coreos/platforms/iso/).
+1.  Once you've downloaded the ISO image, boot your Kubernetes master from the image.  Once booted, you should be automatically logged in as the `core` user at the terminal. At this point CoreOS is running from the ISO and it hasn't been installed yet.
```

Comment:
Booting from an ISO is not bare metal.

Say - Either burn the iso to a CD/DVD/USB key and boot from it, or if using a virtual machine boot directly from it.